package.json

```diff
@@ -8,7 +8,9 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "postinstall": "cd ./ios && pod install && cd .."
+    "postinstall": "cd ./ios && pod install && cd ..",
+    "clean:android": "cd ./android && ./gradlew clean && cd ..",
+    "clean:ios": "cd ./ios && xcodebuild clean && cd .."
```

Comment:
I made clean script, but `clean:ios` is not working in my Mac (using M1 chip).
I'll follow up this issue later.
If you need to clean ios build, just remove `ios/build` folder manually.